ngraph/test/CMakeLists.txt

```diff
@@ -508,14 +508,46 @@ if (NGRAPH_ONNX_EDITOR_ENABLE)
         onnx/onnx_import_with_editor.in.cpp)
 endif()
 
-# SOURCE AND HEADERS FOR FRONTEND TESTING
+# SOURCE FOR FRONTEND TESTING
+
 file(GLOB FRONTEND_TESTS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/frontend/frontend_manager.cpp)
 set(SRC ${FRONTEND_TESTS_SRC} ${SRC})
 
 file(GLOB FRONTEND_SHARED_TESTS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/frontend/shared/src/*.cpp)
 file(GLOB FRONTEND_SHARED_TESTS_HDR ${CMAKE_CURRENT_SOURCE_DIR}/frontend/shared/include/*.hpp)
 set(SRC ${FRONTEND_SHARED_TESTS_SRC} ${SRC})
 
+# ---- PaddlePaddle FrontEnd testing ------
+if (NGRAPH_PDPD_FRONTEND_ENABLE)
+    find_package (PythonInterp 3 REQUIRED)
```

Comment:
Shouldn't we use here at first python passed by `-DPYTHON_EXECUTABLE=`?

Reply:
This way should take -DPYTHON_EXECUTABLE correctly, please see this discussion https://github.com/openvinotoolkit/openvino/pull/6036#discussion_r656008155

Reply:
Ok, I see, I've tested locally the version before this fix

Reply:
@mbencer great, can you please approve this PR then? You've "requested changes" and merge will be blocked without your approval